test/verify/check-active-pages

```diff
@@ -30,7 +30,7 @@ class TestActivePages(MachineCase):
         self.login_and_go("/system")
 
         # Preloading of playground/preloaded only happens with #12534
-        if m.image in [ "rhel-8-0-distropkg" ]:
```

Comment:
Is there any need to keep `rhel-8-0-distropkg` here?
I am not sure what exactly we want - whether to keep `rhel-8-0-distropkg` but just not to run it automatically or to drop it completely. On one hand you removed it from testmap, so we cannot trigger it and you also removed it from doc, on the other hand you kept the install script and keep it also in tests.

Reply:
Right, I wanted to get this to green, land it, disable 8-0-distropkg, and as a follow-up remove all the references to it an the rhel-8-0 image. I just didn't want to conflate the two.